ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -84,6 +111,34 @@
     }
   }
 
+  &__header-text {
+    margin-left: 3rem;
```

Comment:
This seems to have caused the title to be off centre. I think we need to centre align the search component, but make the text left aligned.

Reply:
removed the margin-left @jlhdxw